### ISSUE TITLE
Unexpectedly large number of paths that slow down what should be "simple"

### ISSUE THREAD

**Issue by 0xalpharush** [crytic/optik#53]
When I run this example with Echidna, it broke all but one assertion, `testNegativeMax1`, very quickly. I expected for Optik to only try to solve this unbroken assertion as it should have counterexamples in the corpus for others, but it seemed "more difficult" of a problem than I imagined. Intuitively, counterexamples would be raise early on for the other assertions (this is understandable given the UI is still in the works). Also, I'd expect `val != type(int256).max - 2` to be fed into an SMT solver and for a value to be found trivially. Just giving my feedback on UX... there's likely something I'm missing as Maat/Optik are a black box to me.
```
contract FuzzNumbersTest {

    function testNegativeHalf(int256 val) public {
        assert(val < 2 ** 128 - 1);
    }

    function testNegative0(int256 val) public {
        assert(val != 0);
    }

    function testNegative1(int256 val) public {
        assert(val != -1);
    }

    function testNegative2(int256 val) public {
        assert(val != 1);
    }

    function testNegativeMax0(int256 val) public {
        assert(val != type(int256).max);
    }

    function testNegativeMax1(int256 val) public {
        assert(val != type(int256).max - 2);
    }

    function testNegativeMin0(int256 val) public {
        assert(val != type(int256).min);
    }


    function testNegativeMin1(int256 val) public {
        assert(val != type(int256).min + 2);
    }

    function testEquality(int256 x, int256 y) public {
        int256 xy;

        unchecked {
            xy = x * y;
        }

        if ((x != 0 && xy / x != y)) return;

        assert(((xy - 1) / 1e18) + 1 == (xy - 1) / (1e18 + 1));
    }
}

```
Optik found many paths, `Solving potential new paths... (2138 total, 604 unique)`, and ran for multiple hours.

**Comment by Boyan-MILANOV** [comment]
I'll have a look. What happens if you add `--cov-mode inst` to the command line?

**Comment by Boyan-MILANOV** [comment]
The new incremental seeding strategy implemented in #56 allows to solve the `testNegativeMax1` quickly :)
